<launch> 
					<!-- Gazebo-->
    <include file="$(find rrt_learn_gazebo)/launch/teresa_gazebo.launch"/>
  <node pkg="tf" type="static_transform_publisher" name="odom_to_rgbd_cam" args="0.0 -2 0.677 0 0 0 odom camera_link 100"/>
  <node pkg="tf" type="static_transform_publisher" name="base_link_to_left_wheel" args="0.0 -0.252 0.0725 0 0 0 base_link left_wheel_link 100"/>
  <node pkg="tf" type="static_transform_publisher" name="base_link_to_right_wheel" args="0.0 0.252 0.0725 0 0 0 base_link right_wheel_link 100"/>

<node name="map_server" pkg="map_server" type="map_server" args="$(find rrt_learn_gazebo)/cfg/map2.yaml">
        <param name="frame_id" value="/map" />
    </node>

  <node pkg="laser_assembler" type="laser_assembler" name="laser_assembler" args="/scanback /scanfront">
  </node>

<!--include file="$(find active_perception_controller)/launch/laser_assemble.launch" /-->
					<!-- AMCL-->
    <include file="$(find rrt_learn_gazebo)/launch/amcl.launch" />

					<!-- RVIZ-->
<node pkg="rviz" name="rviz" type="rviz" args="-d $(find rrt_learn_gazebo)/cfg/sevilla_filter.rviz"/>

				<!--Configuration Changer -->
    <node pkg="active_perception_controller" type="robot_config_change.py" name="config_changer" output="screen">
    </node>  

    					     <!-- RRT -->
    <node pkg="active_perception_controller" type="sevilla_data_filtering.py" name="mmp_learner" output="screen">	
    </node>  		

					<!--People to costmap -->
    <node pkg="active_perception_controller" type="people_to_costmap.py" name="people_costmap" output="screen">
    </node>  
			
</launch>
